arcade/paths.py

```diff
@@ -266,19 +341,23 @@ def astar_calculate_path(start_point: Point,
 def has_line_of_sight(point_1: Point,
                       point_2: Point,
```

Comment:
We may as well rename these to something clearer. Maybe something like `observer` and `target`?